brave/browser-laptop
# added yarn lock file

Fix #6437

Auditors @bsclifton @bbondy 

Test Plan:

1. Install Yarn (https://yarnpkg.com/en/docs/install)
2. check out this branch
3. yarn install
4. yarn run watch-all
5. (in another terminal) yarn start
